<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logIn</class>
 <widget class="QMainWindow" name="logIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>logIn</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="logInButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
